instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py

```diff
@@ -50,6 +50,17 @@ def hello():
     application = get_wsgi_application()
     application = OpenTelemetryMiddleware(application)
 
+
+Usage in Production Server (gunicorn)
+------------------------------------
+
+Modify the ``gunicorn.config.py`` file as shown below.
+
+.. code-block:: python
+
+    def post_fork(server, worker):
+        worker(DjangoInstrumentor().instrument())
```

Comment:
2. Why is result of instrument() being passed on to worker? Is it even expected or required for a worker to be called like this in post_fork? AFAIK, you can totally ignore doing anything with the server and worker object, and just perform general initialization but may be I'm wrong.

Reply:
I think you're right actually. I was thinking about that too but I thought instrumentation should pass through a worker. I see now and agree that passing it into post_fork is quite sufficient

Reply:
I see that you recommended a pipeline. Let me put that in effect